jest.config.js

```diff
@@ -3,7 +3,10 @@ module.exports = {
   testRegex: '((\\.|/)(e2e|test|spec))\\.[jt]sx?$',
   testSequencer: require.resolve('./jest.sequencer'),
   moduleNameMapper: {
-    '@defichain/(?!whale-api-client)(.*)': '<rootDir>/packages/$1/src'
+    '@defichain/(?!whale-api-client)(.*)': '<rootDir>/packages/$1/src',
+
+    // apps
```

Comment:
```suggestion
    // libs
```
Just to be clear non of the `app` component should be importable